<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocialNetworkWindow</class>
 <widget class="QMainWindow" name="SocialNetworkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocialNetworkWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="logEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nameLbl">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>109</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter your name:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="logBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="pfLog">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="frndTbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>351</width>
      <height>192</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>12</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>325</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>32</number>
    </attribute>
    <column>
     <property name="text">
      <string>                     Friends                     </string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>260</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="sugTbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>351</width>
      <height>192</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>12</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>325</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>32</number>
    </attribute>
    <column>
     <property name="text">
      <string>                     Friend Suggestions (Double Tap to Add)                     </string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add User As Friend</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="PostTbl">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>401</width>
      <height>191</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>Recent Posts (click to see full content of post)</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="outBtn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>270</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <zorder>sugTbl</zorder>
   <zorder>logEdit</zorder>
   <zorder>nameLbl</zorder>
   <zorder>logBtn</zorder>
   <zorder>pfLog</zorder>
   <zorder>frndTbl</zorder>
   <zorder>backBtn</zorder>
   <zorder>addBtn</zorder>
   <zorder>PostTbl</zorder>
   <zorder>outBtn</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
